closure-typing/calculator.js

```diff
@@ -1,5 +1,25 @@
-function calculator() {
-  // your code here
+function calculator(a) {
+   let regexp = new RegExp(/\+|\-|\*|\//)
+    let numFirst = +a.split(regexp)[0];
```

Comment:
As I wrote for everyone else, parseInt or parseFloat is better than sideeffects of '+', and explicit cast in particular